src/main/java/Board.java

```diff
@@ -0,0 +1,117 @@
+import java.util.Arrays;
+
+public class Board {
+
+    private Player[][] grid;
+    private int gridSize = 3;
+
+    public Board() {
+        grid = new Player[gridSize][gridSize];
+        for (Player[] row : grid) {
+            Arrays.fill(row, Player.NEITHER);
+        }
+    }
+
+    public Player getSquareValue(int row, int col) {
+        return grid[row][col];
+    }
+
+    public void setSquareValue(int row, int col, Player player) {
+        grid[row][col] = player;
+    }
+
+    public boolean isEmptySquare(int row, int col) {
+        return squareMatches(row, col, Player.NEITHER);
+    }
+
+    public boolean isFull() {
+        for (int row = 0; row < gridSize; row++) {
+            for (int col = 0; col < gridSize; col++) {
+               if (isEmptySquare(row, col)) {
+                   return false;
+               }
+            }
+        }
+        return true;
+    }
+
+    public boolean isAnyWinningRow(Player player) {
```

Comment:
Do these need to be public? I'm guessing no clients of this class really need to know the details of whether it's a row, column or diagonal win.